dev/breeze/src/airflow_breeze/commands/ci_commands.py

```diff
@@ -283,6 +282,8 @@ class WorkflowInfo(NamedTuple):
     pr_number: int | None
 
     def get_all_ga_outputs(self) -> Iterable[str]:
+        from airflow_breeze.utils.github import get_ga_output
```

Comment:
Those local imports are needed for auto-complete of breeze to work. This is a click limitation - we have to parse the click methods when we do auto-completion, but it is done outside of the`pipx` environment, therefore we cannot import third-party packages while importing top-level code of the commands.